src/components/molecules/FeaturedBlog.tsx

```diff
@@ -0,0 +1,115 @@
+import Link from "../atoms/Link";
+import Date from "../atoms/Date";
+import Container from "../atoms/Container";
+import Text from "../atoms/Text";
+import Heading from "../atoms/Heading";
+import Image from "../atoms/Image";
+import { TextSizes } from "../../sharedVariants";
+
+interface FeaturedBlogProps {
+  /**
+   * @displayName Blog Title
+   * @tooltip Controls the title of the blog to display
+   */
+  name?: string;
+  /**
+   * @displayName Blog URL
+   * @tooltip Controls the URL the blog redirects to
+   */
+  slug?: string;
+  /**
+   * @displayName Blog Image URL
+   * @tooltip Source URL of the blog image
+   */
+  src?: string;
+  /**
+   * @displayName Blog Image Alternate Text
+   * @tooltip Alternate text for the blog image
+   */
+  alt?: string;
+  /**
+   * @displayName Blog Description
+   * @tooltip Controls the description to display for the blog
+   */
+  blogStarter_description?: string;
+  /**
+   * @displayName Date Posted
+   * @tooltip Controls the posted date to display for th eblog
+   */
+  datePosted?: string;
+  /**
+   * @displayName Image Shape
+   * @tooltip Controls the shape of the blog image
+   */
+  imgShape: "Rectangle" | "Square";
+  /**
+   * @displayName Date Size
+   * @tooltip Controls the size of the posted date text
+   */
+  dateSize: TextSizes;
+  /**
+   * @displayName Title Rank
+   * @tooltip Controls whether the title is an h1, h2, h3 or h4
+   */
+  titleRank: "1" | "2" | "3" | "4";
+  /**
+   * @displayName Description Size
+   * @tooltip Controls the size of the description text
+   */
+  descriptionSize: TextSizes;
+}
+
+export const initialProps: FeaturedBlogProps = {
+  name: "Blog Post Title",
+  slug: "blog-post-example",
+  src: "http://a.mktgcdn.com/p/G_jLt9S27Dwyvf26th8ZsXNmCqXTgUh9TEuGfJxtkuo/2070x1380.jpg",
```

Comment:
same comment as above regarding image placeholder url

Reply:
done - also updated to the placeholder in my layout